tests/appsec/test_customconf.py

```diff
@@ -57,6 +57,9 @@ def test_requests(self):
         interfaces.library.assert_waf_attack(self.r_2, pattern="dedicated-value-for-testing-purpose")
 
     def test_log(self):
+        """Check if it's implemented for the weblog variant"""
+        if context.library == "java":
+            stdout.assert_presence("AppSec is FULLY_ENABLED with powerwaf")
         """Check there is no error reported in logs"""
```

Comment:
Duplicated doc string. The `"""Doc string"""` should be just one, and at the top of the function.
For further comments on specific lines, use a `# Regular line comment`.